fhir-parent/pom.xml

```diff
@@ -641,6 +641,16 @@
                 <artifactId>azure-core</artifactId>
                 <version>1.24.0</version>
             </dependency>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-stdlib</artifactId>
+                <version>1.6.10</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-stdlib-common</artifactId>
+                <version>1.6.10</version>
+            </dependency>
```

Comment:
maybe a comment indicating these are used in azure blob sdk from fhir-persistence-blob?